commit 329ef2c4ebcb3aec6dcfcac15357fd583a60c969
Author: Mike Gorse <mgorse@suse.com>
Date:   Tue Jan 24 18:10:52 2017 -0600

    atspi_application_dispose: clear root accessible's ref to the app
    
    Hoping that this might fix https://bugzilla.gnome.org/show_bug.cgi?id=767074

diff --git a/atspi/atspi-application.c b/atspi/atspi-application.c
index 65cabdc..f7dd225 100644
--- a/atspi/atspi-application.c
+++ b/atspi/atspi-application.c
@@ -58,6 +58,7 @@ atspi_application_dispose (GObject *object)
 
   if (application->root)
   {
+    g_clear_object (&application->root->parent.app);
     g_object_unref (application->root);
     application->root = NULL;
   }
